### PR TITLE
Add a regression test for VSO 284785.

### PR DESCRIPTION
**Pull request by pgavlin**
Just what it says on the tin.

### CODE REVIEW

**Comment by pgavlin** [comment]
@BruceForstall @adiaaida @dotnet/jit-contrib PTAL.

I meant to add this as part of #7983.

File: tests/src/JIT/Regression/JitBlue/DevDiv_284785/DevDiv_284785.cs [hunk 1]
```diff
@@ -0,0 +1,34 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
```
Comment (BruceForstall) [inline tests/src/JIT/Regression/JitBlue/DevDiv_284785/DevDiv_284785.cs]: 1. It is useful to have a comment stating what is intended to be tested, e.g., what failed before and why.
2. I think the standard failure result code is 101, not zero.

**Comment by BruceForstall** [comment]
LGTM modulo comments

File: tests/src/JIT/Regression/JitBlue/DevDiv_284785/DevDiv_284785.cs (hunk 1 above)
Comment (pgavlin) [inline tests/src/JIT/Regression/JitBlue/DevDiv_284785/DevDiv_284785.cs]: There is no standard failure code--anything other than 100 will do.

Comment (BruceForstall) [inline tests/src/JIT/Regression/JitBlue/DevDiv_284785/DevDiv_284785.cs]: There is no *required* failure code, I would say there is a *standard* code, or at least a commonly used failure code. IMO, zero is too easily misinterpreted.

**Pull request merged**